app.py

```diff
@@ -50,7 +51,21 @@ def _get_local_time():
             return f"Could not process time information. {e}", 404
 
 
-#TODO Add a route that displays the current time for all US timezones.
+@app.route('/time-difference', methods=['GET', 'POST','PUT'])
```

Comment:
I'm confused about the multiple request methods, should they all be accepted here?